.github/pull_request_template.md

```diff
@@ -19,9 +17,16 @@ Delete an option.
 - 🚨 Yes, this touches <flow>
 - ✅ Nope
 
-
 ### How Has This Been Tested?
 
+Tests Checklist:
```

Comment:
I don't think this is the right direction to take this tbh. I would rather see us incorporate CI correctly, which already displays in github, e.g. mad dog can be run and should show passing in github.

Not all changes require client "work" or changes there, and unit tests should also be captured in CI.

Reply:
that's fair -- I was hoping that this checklist acts as a reminder to verify parts of the audius protocol/client works as expected VS. necessary steps that need to be taken

Reply:
will close for now then

Reply:
Not opposed to making more specific checklists to help us here. Would be great if there was a better way to orchestrate that with this monorepo. E.g. libs changes should definitely be tested with the client + should probably have a successful probers run happen. First step there is probably make the probers 100% reliable